theme/icons/strikethrough.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="1.414"><path d="M8.396 10.861c1.83.02 3.66.038 5.489.014.311.54.466 1.147.466 1.823 0 .944-.373 1.834-1.119 2.671-.745.837-1.94 1.255-3.584 1.255a6.429 6.429 0 0 1-2.291-.429l-.366-.137a2.901 2.901 0 0 0-.246-.059 1.113 1.113 0 0 0-.201-.023c-.195 0-.329.056-.402.169a1.704 1.704 0 0 0-.192.479h-.538v-4.629h.538c.305 1.394.796 2.408 1.475 3.041s1.456.949 2.333.949c.852 0 1.458-.217 1.817-.653.359-.435.539-.893.539-1.374 0-.56-.189-1.016-.566-1.369-.25-.232-.752-.533-1.507-.904l-1.214-.603a15.578 15.578 0 0 1-.431-.221zm2.881-2.034l.535.264c.459.228.85.473 1.172.736H6.806a4.395 4.395 0 0 1-.527-.516 2.98 2.98 0 0 1-.334-.484h5.332zm2.526-1.379h-.521a5.656 5.656 0 0 0-1.319-2.438c-.63-.682-1.395-1.023-2.296-1.023-.633 0-1.134.172-1.502.516a1.653 1.653 0 0 0-.553 1.256c0 .59.174 1.032.521 1.324.194.169.523.385.987.645-1.177.006-2.355.023-3.532.053a4.065 4.065 0 0 1-.048-.634c0-.913.327-1.766.981-2.557.654-.791 1.657-1.187 3.009-1.187a6.46 6.46 0 0 1 1.995.31c.642.207 1.009.311 1.1.311.207 0 .35-.059.429-.178a1.53 1.53 0 0 0 .192-.47h.557v4.072z" fill="#454545"/><path fill="#454545" d="M4 8.827h12v1H4z"/></svg>
```

Comment:
@Natim just to confirm this is a different icon now right? not the one you got from the internet? 😄

Reply:
Yes that's @oleq one.